php/testSQL.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+mysql_connect("127.0.0.1", "root", "25");
+mysql_select_db("test");
+mysql_query("CREATE TABLE Students (
+    StudId INT NOT NULL PRIMARY KEY AUTO_INCREMENT, 
+    StudName VARCHAR(20)
+    );");
+
+mysql_query("INSERT INTO Students (StudName) VALUES ('Germaine Tailleferre');");
+mysql_query("INSERT INTO Students (StudName) VALUES ('Louis Durey');");
+mysql_query("INSERT INTO Students (StudName) VALUES ('Darius Milhaud');");
+mysql_query("INSERT INTO Students (StudName) VALUES ('Arthur Honegger');");
+mysql_query("INSERT INTO Students (StudName) VALUES ('Georges Auric');");
+mysql_query("INSERT INTO Students (StudName) VALUES ('Francis Poulenc');");
+
+mysql_query("CREATE TABLE Books (
+BookId INT NOT NULL, 
+BookName VARCHAR(20),
+StudId INT(10)
+);");
+
+mysql_query("INSERT INTO Books (BookId, BookName, StudId) VALUES (1, 'Notes', 1);");
+mysql_query("INSERT INTO Books (BookId, BookName, StudId) VALUES (2, 'Sol-Fa', 1);");
```

Comment:
Could you rewrite it using three tables students, books, and relations between them?